site/pages/rest-api/examples/update-role.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Update Role
+---
+
+This command will POST the file `role.xml` (can be a partial XML) to the server to update an existing role.
+
+[Sample Role XML File](./role.xml)
```

Comment:
This can be removed, as we currently don't have a way to display raw files right now.